charactersheet/charactersheet/utilities/fixtures.js

```diff
@@ -1,6 +1,72 @@
 'use strict';
 
 var Fixtures = {
+    defaultProfilePictures: [
+        {
+            name: '',
+            image: '/images/sample-headshots/elf-head.png'
```

Comment:
Can we make this the full quality link? That way it looks like a link in the modal.